### ISSUE TITLE
Auto load last used model on UI startup

### ISSUE THREAD

**Issue by raymerjacque** [lmstudio-ai/lmstudio-bug-tracker#191]
Can you please make it Auto load the last used model when the UI starts up, or atleast have the option to do so ? its so tiring having to open the ui, then click to load the model every time. 99% of people tend to use the same model they last used, so why does it not auto load ? or at least have the option to auto load ? this is in GUI mode.

**Comment by adnovea** [comment]
+1 
Good feature to implement.

**Comment by Rastafabisch** [comment]
I was searching for that feature as well. I could not imagine it wasn't there.

**Comment by pguyennet** [comment]
+1

**Comment by raymerjacque** [comment]
Seems to fall on deaf ears.... I asked about this in their discord a while back and never even got a reply, half the mods and devs were in the channel. I am the owner of the MakuluLinux project, we get around anywhere between  5k to 10k downloads a week and I wanted to include LM studio with a preloaded model into the distro. So user need only open the app and model loads... Make it automated in options so use can just open app, start chatting. But no one seems to want to add the feature and I don't have time to go dig through this code and add it myself, I don't like meddling in other people's code.  To date we are one of the very few Linux distros that come with a lot of built in AI, this would have been a nice addition.

**Comment by erasmion** [comment]
strange that no such option exists

**Comment by raymerjacque** [comment]
Indeed. Anyway, I guess it's a blessing in disguise. It forced me to write my own offline AI model that loads everytime it's opened... I no longer use LM studio, nor do I feature it in the distro. In fact this feature request can be closed.

**Comment by erasmion** [comment]
good for you, being on windows and not being a wizard, i was still able to get it done with some autohotkey 

since i named the models my way, i now have a menu able to open any predefined model or the last used

surprised how well it works

**Comment by joebnb** [comment]
this still not implement as a llm server there wasn't have a failure recovery that i can't imagine

**Comment by yagil** [comment]
You can leverage the lms CLI to do this, specifically [`lms load`](https://lmstudio.ai/docs/cli/load). If you want this to happen on machine start up, use whichever script mechanism your operating system supports. Otherwise if it's a part of you own code, consider either spawning `lms` or using one of our SDKs. See more: https://lmstudio.ai/docs/app/api

**Comment by joebnb** [comment]
> You can leverage the lms CLI to do this, specifically [`lms load`](https://lmstudio.ai/docs/cli/load). If you want this to happen on machine start up, use whichever script mechanism your operating system supports. Otherwise if it's a part of you own code, consider either spawning `lms` or using one of our SDKs. See more: https://lmstudio.ai/docs/app/api

so what is the benefit for use GUI present software instead Ollama

**Comment by yagil** [comment]
You had asked about "llm server", so a programatic way as I mention above could be suitable. LM Studio ships with a GUI, SDK, and CLI in one package. You can choose to use whichever interface suits your needs.
